zammad/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: zammad
-version: 10.0.5
+version: 10.0.6
```

new feature should raise minor version :)

```suggestion
version: 10.10
```